### ISSUE TITLE
Error building the package

### ISSUE THREAD

**Issue by konrad-gajewski** [ornl-epics/dbwr#8]
Hi,
I am trying to build a package using maven but it fails with the following error:
kgajewski@freia-dbwr dbwr]$ mvn package
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Display Builder Web Runtime 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.5:resources (default-resources) @ dbwr ---
[debug] execute contextualize
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.3.2:compile (default-compile) @ dbwr ---
[INFO] Compiling 61 source files to /home/kgajewski/dbwr/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Failure executing javac, but could not parse the error:
javac: invalid target release: 15
Usage: javac <options> <source files>
use -help for a list of possible options

[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.143s
[INFO] Finished at: Thu Feb 17 17:25:51 CET 2022
[INFO] Final Memory: 7M/150M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project dbwr: Compilation failure
[ERROR] Failure executing javac, but could not parse the error:
[ERROR] javac: invalid target release: 15
[ERROR] Usage: javac <options> <source files>
[ERROR] use -help for a list of possible options
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

Trying -e and -X switches did not help me to solve the problem.
I'm trying to build the package and run it on Centos 7.7 (tomcat 7)

[kgajewski@freia-dbwr dbwr]$ mvn -v
Apache Maven 3.0.5 (Red Hat 3.0.5-17)
Maven home: /usr/share/maven
Java version: 1.8.0_232, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.232.b09-0.el7_7.x86_64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-1062.el7.x86_64", arch: "amd64", family: "unix"

Thanks,
Konrad

**Comment by kasemir** [comment]
The current version of Java is 17.
We require at least Java 15 (`javac: invalid target release: 15`).
You're using 8 (`Java version: 1.8`)

**Comment by kasemir** [comment]
See http://jdk.java.net/

**Comment by konrad-gajewski** [comment]
I have updated to Java 17 and maven 3.8.4
[kgajewski@freia-dbwr dbwr]$ mvn -v
Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)
Maven home: /opt/maven
Java version: 17.0.1, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-17-openjdk-17.0.1.0.12-1.rolling.el7.x86_64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-1062.el7.x86_64", arch: "amd64", family: "unix"

Still failed to build - see attached log.

[dbwr-install.log](https://github.com/ornl-epics/dbwr/files/8092331/dbwr-install.log)

**Comment by yxrmz** [comment]
ok, I modified the pom.xml following [this answer](https://stackoverflow.com/a/68914243/18239635). Here's my pom
[pom.xml.txt](https://github.com/ornl-epics/dbwr/files/8093809/pom.xml.txt)
[INFO] BUILD SUCCESS

**Comment by konrad-gajewski** [comment]
Thanks @yxrmz . I have modified pom.xml according to your hints and it worked.
BTW, after installing Java 17 making it a default version (alternatives --config java) was not enough. I had to set JAVA_HOME to point to Java 17 to be used by maven. I managed to build the package with both maven version 3.8.4 and original 3.0.5 from Centos7 repository. 

@kasemir , will you update the repository or is it too specific to Centos7 and Ubuntu20.04?
